### PR TITLE
3837 force export related

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

**Comment by abea** [comment]
What does `lead` mean in this context?

File: views/force-export-modal.html [hunk 1]
```diff
@@ -20,6 +20,16 @@
   <p>
     {{ __('You are forcing an export, which will copy this draft verbatim to other locales. Select them below, then click Export. Selecting the %s locale may still be helpful to select sub-locales.', data.doc.workflowLocale, data.doc.workflowLocale) }}
   </p>
+  {% if not data.lead %}
+    <p>
+      {{ __('This related item may also be visible. Please consider exporting it too for a more complete result.') }}
+    </p>
+    <p>
+      {# Despite the data attribute name this will do the right operation here #}
```
Comment (abea) [inline views/force-export-modal.html]: Maybe drop a TODO here at least to mark that the naming should be updated in a later version?

**Comment by boutell** [comment]
I see your point but I'm going to let it lie because it would be inconsistent with the naming for the same api parameter in the commit workflow at this point.

**Comment by boutell** [comment]
This name will get fixed in 3.x which will have to have an entirely new UI build anyway (because Vue).

**Pull request merged**